lib/DeploymentProvider/Providers/WebAppDeploymentProvider.js

```diff
@@ -75,5 +76,14 @@ class WebAppDeploymentProvider extends BaseWebAppDeploymentProvider_1.BaseWebApp
             }
         });
     }
+    UpdateDeploymentStatus(isDeploymentSuccess) {
```

Comment:
Why is this method not present in the typescript file?

Reply:
It is there. https://github.com/Azure/webapps-deploy/blob/a98be40ccf5d3017a21681529ff244f96d597fca/src/DeploymentProvider/Providers/WebAppDeploymentProvider.ts#L72